requirements.txt

```diff
@@ -1,7 +1,6 @@
 Cerberus==0.9.2
-Eve==0.7.4
+eve == 0.7.5
```

Comment:
This will conflict with #201, which updates Eve to v0.8

Reply:
reverted